types/braintree/braintree-tests.ts

```diff
@@ -109,5 +108,6 @@ const gateway: BraintreeGateway = new braintree.BraintreeGateway({
     const notificationResponse = await gateway.webhookNotification.parse(sampleResponse.bt_signature, sampleResponse.bt_payload).catch(console.error);
     if (!notificationResponse) return;
 
-    const notification: WebhookNotification = notificationResponse;
+    const notification: SubscriptionNotification = <SubscriptionNotification> notificationResponse;
```

Comment:
I believe the preference it to cast a type using the `as` syntax (https://www.typescriptlang.org/docs/handbook/release-notes/typescript-1-6.html#new-tsx-file-extension-and-as-operator).